.github/workflows/publish_lambda_layer.yaml

```diff
@@ -0,0 +1,99 @@
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+name: Publish Lambda Layer to staging
+
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        description: 'Name of actions/upload-artifact result'
+        required: true
+        type: string
+      component-version:
+        description: 'Version of APM Python library in this release'
+        required: true
+        type: string
+      architecture:
+        description: 'Must be x86_64 or arm64'
+        required: true
+        type: string
+      runtimes:
+        description: 'Space-delimited list of compatible runtimes'
+        required: true
+        type: string
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
Are these permissions actually needed?

Reply:
Nope not needed 👍 Removed in [0b10ae7](https://github.com/solarwinds/apm-python/pull/221/commits/0b10ae7dab693b3634e5913c1619dd95146b4e55), tested in https://github.com/solarwinds/apm-python/actions/runs/6879510772/job/18711579223?pr=221